google_fonts/lib/src/google_fonts_base.dart

```diff
@@ -20,26 +25,68 @@ http.Client httpClient = http.Client();
 @visibleForTesting
 void clearCache() => _loadedFonts.clear();
 
-/// Loads a font into the [FontLoader] with [fontName] for the matching
-/// [fontUrl].
+/// Creates a [TextStyle] that either uses the font family for the requested
+/// GoogleFont, or falls back to the pre-bundled font family.
+///
+/// This function has a side effect of loading the font into the [FontLoader],
+/// either by network or from the file system.
+TextStyle googleFontsTextStyle({
+  @required String fontFamily,
+  TextStyle textStyle,
+  FontWeight fontWeight,
+  FontStyle fontStyle,
+  @required Map<GoogleFontsVariant, String> fonts,
+}) {
+  assert(fontFamily != null);
+  assert(fonts != null);
+
+  textStyle ??= TextStyle();
+  textStyle = textStyle.copyWith(fontWeight: fontWeight, fontStyle: fontStyle);
+
+  final variant = _closestMatch(
+      GoogleFontsVariant(
+        fontWeight: textStyle.fontWeight ?? FontWeight.w400,
+        fontStyle: textStyle.fontStyle ?? FontStyle.normal,
+      ),
+      fonts.keys);
```

Comment:
nit:

```suggestion
      fonts.keys,);
```

And run dartfmt